src/main/res/layout/profile_preference_view.xml

```diff
@@ -29,12 +29,14 @@
                 android:textSize="20sp"
                 tools:text="Voltarine DeClyre"/>
 
-        <TextView android:id="@+id/number"
+        <TextView android:id="@+id/profile_status"
                   android:layout_width="match_parent"
                   android:layout_height="wrap_content"
                   android:gravity="start"
                   android:textAlignment="viewStart"
-                  tools:text="+14151231234"/>
+                  android:singleLine="true"
+                  android:ellipsize="marquee"
```

Comment:
wondering, why we're not using `android:ellipsize="end"` here.

however, grepping for "marquee", we're using that at lots of places - but i have never seen sth scrolling (which is good i'd say) - so, as it is done as elsewhere, probably fine :)